.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 vendor
+*.lock
 test/coverage/*
 dist/
 .env*
-smtpapi-php.php
```

Comment:
We ignore this one because it's only needed for the [installation from zip](https://github.com/sendgrid/smtpapi-php#alternative-install-from-zip). But, maybe no harm in leaving it in, for those who clone the repo.

Reply:
It's not ignored in `sendgrid-php`: https://github.com/sendgrid/sendgrid-php/blob/master/.gitignore